openslides_backend/shared/mixins/user_create_update_permissions_mixin.py

```diff
@@ -382,12 +401,26 @@ def check_group_F(
                 )
             else:
                 if self.permstore.user_committees.intersection(
-                    self.instance_committee_ids
+                    self.instance_committee_meeting_ids
                 ):
                     return
                 expected_oml_permission = OrganizationManagementLevel.CAN_MANAGE_USERS
-            if expected_oml_permission > self.permstore.user_oml:
-                raise MissingPermission({expected_oml_permission: 1})
+            if not (
+                expected_oml_permission <= self.permstore.user_oml
+                or self.check_for_admin_in_all_meetings(instance.get("id", 0))
+            ):
+                raise MissingPermission(
+                    {
+                        expected_oml_permission: 1,
+                        **{
+                            Permissions.User.CAN_UPDATE: meeting_id
+                            for meeting_ids in self.instance_committee_meeting_ids.values()
+                            if meeting_ids is not None
+                            for meeting_id in meeting_ids
+                            if meeting_id is not None
```

Comment:
This is just going to return the last meeting that was checked where the permission is missing.
Always using the same keyword means it will be overwritten every time, after all.
Also: Missing permission concatenates the permissions with 'or' and never 'and'.
If that is what you wanted, it's fine.
If it is not, you should probably looking into using something other that `MissingPermission` and generating your own response message. I've seen the `MissingPermission` base class used like that before.